src/surface/surface.scss

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-surface($background-color: var(--iui-color-background-1), $border-radius: 0, $elevation: $iui-elevation-2) {
+  @include surface($background-color, $border-radius);
+  @include elevation($elevation);
+}
+
+@mixin surface($background-color: var(--iui-color-background-1), $border-radius: 0) {
+  --iui-surface-background-color: #{$background-color};
+  --iui-surface-border-radius: #{$border-radius};
```

Comment:
Maybe remove for now as we don't have CSS variable for that and we only use one value (no?).

Reply:
Still doesn't hurt to keep, as you can set any value from outside, e.g. `--iui-surface-border-radius: 12px` via react prop `<Surface borderRadius='12px' />`